pkg/verification/verify.go

```diff
@@ -75,3 +254,18 @@ func verifyHashedMessages(hashAlg hash.Hash, hashedMessage []byte, artifactReade
 
 	return nil
 }
+
+func CreateTimestampResponse(tsrBytes []byte) (timestamp.Timestamp, error) {
```

Comment:
Is this needed in verify?

Reply:
No. It looks like the function isn't used anywhere, so I'll remove it.`VerifyTimestampResponse` will parse the bytes and check TSR status.